### PR TITLE
Select a card endpoint

### PR DESCRIPTION
**Pull request by kezc**
Added a new endpoint for selecting a card

### CODE REVIEW

File: backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/CustomizedResponseEntityExceptionHandler.kt [hunk 1]
```diff
@@ -37,4 +38,13 @@ class CustomizedResponseEntityExceptionHandler : ResponseEntityExceptionHandler(
...
         return ResponseEntity<ExceptionResponse>(exceptionResponse, HttpStatus.CONFLICT)
     }
+
+    @ExceptionHandler(PlayerDoesNotExistException::class)
+    fun handleUsernameTakenException(
+        playerDoesNotExist: PlayerDoesNotExistException,
+        request: WebRequest
+    ): ResponseEntity<ExceptionResponse> {
+        val exceptionResponse = ExceptionResponse(playerDoesNotExist.message.orEmpty())
+        return ResponseEntity<ExceptionResponse>(exceptionResponse, HttpStatus.NOT_ACCEPTABLE)
```
Comment (kezc) [inline backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/CustomizedResponseEntityExceptionHandler.kt]: is 406 a good response for that... I don't want to reply with 404 as it's used for no such room inside the same endpoint

Comment (Zuretto) [reply backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/CustomizedResponseEntityExceptionHandler.kt]: I'd go with 404 and exception-specific exceptionResponse (you can use Exception's message to instantiate it during creation of Exception), but if you feel it's too much effort then it's acceptable...

**Review by Zuretto — DISMISSED** [review]
good job, please see my answer to your comment

**Review by Zuretto — APPROVED** [review]
Thanks!

**Pull request merged**